src/components/Notepad/Notepad.tsx

```diff
@@ -87,10 +97,11 @@ export default function Notepad({ id, title }: { id: string; title: string }) {
           <ReactQuill
             modules={{
               toolbar: [
-                [{ header: [1, 2, 3, false] }],
+                [{ font: Font.whitelist }],
+                [{ size: ["18px", "28px", "38px"] }],
                 ["bold", "italic", "strike"],
                 [{ list: "ordered" }, { list: "bullet" }],
-                // ["video"],
+                ["image"],
```

Comment:
This enables image "upload" functionality which you forgot to mention in your commit message @liam-k. Pretty fucking unprofessional. But it well out of the box with state management & local storage, which means SOMEONE did a good job at least.